### ISSUE TITLE
Failed to watch *unstructured.Unstructured: the server does not allow this method on the requested resource

### ISSUE THREAD

**Issue by matihost** [operator-framework/operator-sdk#1123]
When ansible operator playbook creates RoleBinding objects:
```
{"level":"info","ts":1550486474.1879833,"logger":"logging_event_handler","msg":"[playbook task]","name":"alaMakota","namespace":"","gvk":"ala.cat.com/v1, Kind=GrumpyCat","event_type":"playbook_on_task_start","job
":"7492815135160941009","EventData.Name":"project-apply : Apply RBAC for project alaMaKota project"}
ERROR: logging before flag.Parse: E0218 10:41:14.934850       1 reflector.go:251] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: Failed to watch *unstructured.Unstructured: the server does not allow this method on the requested resource
{"level":"info","ts":1550486475.2151556,"logger":"runner","msg":"Ansible-runner exited successfully","job":"7492815135160941009","name":"alaMaKota-cr","namespace":""}
ERROR: logging before flag.Parse: E0218 10:41:15.965672       1 reflector.go:251] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: Failed to watch *unstructured.Unstructured: the server does not allow this method on the requested resource
ERROR: logging before flag.Parse: E0218 10:41:17.000215       1 reflector.go:251] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: Failed to watch *unstructured.Unstructured: the server does not allow this method on the requested resource
ERROR: logging before flag.Parse: E0218 10:41:18.038124       1 reflector.go:251] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: Failed to watch *unstructured.Unstructured: the server does not allow this method on the requested resource
(... and it repeats every second forever ...)
```
The operator pod service account is essentially cluster-admin : (a a workaround for another case..)
```
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: ala-has-a-cat-operator
rules:
- apiGroups:
  - '*'
  resources:
  - '*'
  verbs:
  - '*'
```

**Comment by shawn-hurley** [comment]
@matihost Is there some time that we could jump on a call and discuss your use cases. Seems like we are not doing as good of a job supporting them as I would like, but I don't really understand what it is your solving. @fabianvf and I work in EST, hopefully, we can find a time that works.

**Comment by matihost** [comment]
I will contact with @fabianvf  via different channel.
This is what I do, not very complex:

- `operator-sdk new some-operator --cluster-scoped --api-version=ala.has.a.cat.com/v1alpha1 --kind=Cat --type=ansible --skip-git-init --generate-playbook`
- change deploy/crd/....crd.yaml:scope from `Namespaced` to `Cluster`
- cjange role.yaml to essentially cluster-admin (like shown above)
- inject `oc/kubectl` command into Dockerfile
- I do the following steps in playbook.yaml:

```
# when project is not created yet
oc new-project {{meta.name}}
oc policy add-role-to-user admin  {{ someVariableFromCR }} -n  {{meta.name}}
```
In other words I create project/namespace - and my ServiceAccount is admin automatically in that project (so essentially cluster-role is not needed - but since CRD is Cluster- the operator needs ClusterRoleBinding on objects )

Afterwards I modify, create rolebinding.
After that line the error is appearing on stdout.

**Comment by matihost** [comment]
It seems it is related to transient ProjectRequest and Project object coming from OpenShift.
